profiler/README.md

```diff
@@ -1,9 +1,46 @@
 
-The profiler is in its infancy and is disabled by default.
+🚨 The profiler is in beta and its use is currently restricted.
```

Comment:
```suggestion
> **Warning** The profiler feature is experimental.
```

Reply:
Yeah, I'll go with that construction bit that @mateuszrzeszutek noted above.